tests/v2_validation/cattlevalidationtest/core/test_ldap_auth.py

```diff
@@ -431,6 +477,10 @@ def test_ldap_create_new_env_add_group_member(admin_client):
         project.setmembers(members=new_members)
     assert "object has no attribute" in str(excinfo.value)
 
+    unset_group_search_domain(admin_client)
+    assert len(main_client.list_identity(name=group)) == 0
+    set_group_search_domain(admin_client)
+
```

Comment:
@prachidamle at first the domain is set `ou=dev,dc=us-west-2,dc=compute,dc=internal` and groupSearchDomain is set to `ou=groups,dc=us-west-2,dc=compute,dc=internal`. This way it's not possible to add the `validationGroup` if the groupSearchDomain field doesn't work. The tests pass so that verifies that the group search works. Then unset() makes it empty, which means no groups should be available. Then for the following tests the groupSearchDomain is again set to the original value